sabnzbd/growler.py

```diff
@@ -539,7 +545,57 @@ def send_pushbullet(title, msg, gtype, force=False, test=None):
             return T('Failed to send pushbullet message')
     return ''
 
+def send_nscript(title, msg, gtype, force=False, test=None):
+    if test:
+        script = test.get('nscript_script')
+    else:
+        script = sabnzbd.cfg.nscript_script()
+    if not script:
+        return T('Cannot send, missing required data')
+    title = u'SABnzbd: ' + Tx(NOTIFICATION.get(gtype, 'other'))
+
+    prio = False
+
+    if gtype == 'startup':
+        prio = sabnzbd.cfg.nscript_prio_startup()
+    if gtype == 'download':
+        prio = sabnzbd.cfg.nscript_prio_download()
+    if gtype == 'pp':
+        prio = sabnzbd.cfg.nscript_prio_pp()
+    if gtype == 'complete':
+        prio = sabnzbd.cfg.nscript_prio_complete()
+    if gtype == 'failed':
+        prio = sabnzbd.cfg.nscript_prio_failed()
+    if gtype == 'disk-full':
+        prio = sabnzbd.cfg.nscript_prio_disk_full()
+    if gtype == 'warning':
+        prio = sabnzbd.cfg.nscript_prio_warning()
+    if gtype == 'error':
+        prio = sabnzbd.cfg.nscript_prio_error()
+    if gtype == 'queue_done':
+        prio = sabnzbd.cfg.nscript_prio_queue_done()
+    if gtype == 'other':
+        prio = sabnzbd.cfg.nscript_prio_other()
+    if force:
+        prio = True
+
+    if prio:
```

Comment:
You don't need any of all this prio stuff, that's only for Prowl/Pushbullet etc. that support notification priorities. 
Look for example at the one for Windows notifications, all you need to do is call the script directly:
```python
def send_windows(title, msg, gtype):
    if sabnzbd.WINTRAY:
        sabnzbd.WINTRAY.sendnotification(title, msg)
    return None
```